app/services/youtube_service.rb

```diff
@@ -4,6 +4,12 @@ def video_info(id)
     get_json('youtube/v3/videos', params)
   end
 
+  def playlist_items(playlist_id, next_page_token = nil)
```

Comment:
Refactor opportunity to increase the results per page to reduce the number of API calls